xcode/Subconscious/Shared/Components/Detail/UserProfileDetailView.swift

```diff
@@ -288,7 +294,17 @@ struct UserProfileDetailModel: ModelProtocol {
             model.user = content.profile
             model.statistics = content.statistics
             model.recentEntries = content.recentEntries
-            model.following = content.following
+            model.following = content.following.map { f in
+                if let petname = f.user.address.petname,
+                   model.pendingFollows.contains(petname) {
+                    
+                    var user = f
+                    user.resolutionStatus = .pending
+                    return user
+                }
```

Comment:
I am not in love with this, but the model is the right place to track pending follows imo.

I experimented with tracking them in the `AddressBookService` but that introduces some nuanced sequenced update problems, since the model ultimately needs access to the pending list to render the UI.

I think we might want to move this up to an app-level concern ultimately? All these background processes are owned the by profile view which is discarded if we pop the navigation stack.